src/profile/scripts/directives.js

```diff
@@ -24,7 +24,9 @@
                             scope.fields = response.data.actions.PUT;
                         });
 
-                    scope.editProfile = function () {
+                    scope.editProfile = function (localStorageVar) {
```

Comment:
I don't really like the way the different modules have a lot repetition. I
know they are meant to be self contained, but if they are being compiled
for distribution anyway, maybe it would make sense to refactor and DRY the
common functionality?

On 21 May 2015 at 13:29, Perry Roper <notifications@github.com> wrote:

> In src/profile/scripts/directives.js
> <https://github.com/incuna/angular-user-management/pull/12#discussion_r30796605>
> :
>
> > @@ -24,7 +24,9 @@
> >                              scope.fields = response.data.actions.PUT;
> >                          });
> >
> > -                    scope.editProfile = function () {
> > +                    scope.editProfile = function (localStorageVar) {
>
> Using localStorage in a name of an object that's not accessing
> Window.localStorage can be confusing. Perhaps something like scopeObject.
> That way it'll give a clue to where the argument is used, and the type.
> Naming things is hard!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/incuna/angular-user-management/pull/12/files#r30796605>
> .
>